.github/workflows/merge-main-v1.yaml

```diff
@@ -25,7 +25,10 @@ jobs:
 
       # run merge without a commit (will be added in final step)
       - name: Run git merge
-        run: git merge --no-commit -s ours origin/v1
+        run: |
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+          git merge -s ours origin/v1
```

Comment:
should we deifne better commit message then the default: `Merge remote-tracking branch 'origin/v1'`

Reply:
qq: is this needed because of bogus version conflict resolution? (1.x vs 2.x version difference in pom.xml)?

Reply:
and yes, we could probably use better message, indicating something like 

"v1->main merge from GH action by [user]"

if possible?

Reply:
by [user]? who is the user?

Reply:
User who triggered the workflow. If possible to determine.

Reply:
Impossible. Plus the main purpose of this is to react on the event of the SG v1 release and create merge automatically.. So this can not be done.

Any other proposal?

Reply:
Ok yeah if not doable then can't do.

Mostly I'd just like to know these PRs came from workflow action -- so whatever static description is fine.
Even the original one is ok, it has enough finrmation.